submissions/oluwatosinOke/html-forms/contactform.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+     <h1> Contact Form </h1>
```

Comment:
`<h1>` and `<form>` tags are not tags that can be found on the `<head>` section. Tags placed inside the `<head>` tag are `<meta>, <link>, <title>, <script>`. So, kindly correct yourself